web/src/shared/components/document-pages/document-pages.tsx

```diff
@@ -1,24 +1,25 @@
 // temporary_disabled_rules
 /* eslint-disable @typescript-eslint/no-redeclare, react-hooks/exhaustive-deps */
-import React, { Fragment, useEffect, useState, useRef, useCallback, useMemo } from 'react';
+import { Spinner } from '@epam/loveship';
```

Comment:
Maybe you could revert unnecessary changes and use the project's ordering of imports (in code style) in order not to format a lot of code.

Reply:
As I can see `prettier` in pre-commit requires such changes. Don't know who and how enabled it, but I just follow existing rules